### DISCUSSION TITLE
Sky & viewing options - DSO preferences

### DISCUSSION THREAD

**Discussion by Xio1996** [Stellarium/stellarium#3354] [Q&A]
Hi Everybody,

Can I check my understanding? 

On the DSO tab of the Sky and viewing options it is possible to select display objects from catalogues and store the preference. This works great and it is possible to switch between the standard and preference catalogues. I then 'Save Settings' but the preference catalogues are not saved across Stellarium sessions. 

Am I missing a step to preserve the settings or would it be possible to add a feature request to store the preferences across sessions?

Many thanks for your help.

Pete

**Comment by gzotti** [comment]
Works well for me. Did you press "store preference" right below the catalogs?

**Reply by Xio1996** [reply]
Hi,

Thank you @gzotti . I think I know what is happening.

If I have switched to standard catalogues on Sky & viewing Options (DSO) and click Save Settings in the Configuration window then the preference catalogues are lost. If I don't press Save Settings then the preferences are stored between sessions, regardless if I have standard or preferences selected.

I have a habit of clicking on Save Settings, just in case! However, I'm not sure Save Settings should overwrite the preference catalogues with the standard catalogues.

Do you see the same behaviour? As this could be a local issue to me.

Many thanks

Pete

**Reply by gzotti** [reply]
Without testing, I think "Save Settings" does not save those catalog settings. Only the buttons local on this page does. 
I am myself no friend of the global "Save Settings" and have started a process of allowing immediate storing of options. However this may not be public before 23.4.

**Reply by Xio1996** [reply]
The immediate save of options rather than a global save is a much better solution. Thank you for such a great application and its continued development.

I'm not sure what is going on with the Catalogue selection. I'm pretty sure the only difference between working and not working was the Save Settings. I will explore a little more.

Many thanks for your help.

Pete
